admin_manual/enterprise/external_storage/windows-network-drive_configuration.rst

```diff
@@ -293,6 +304,75 @@ On the one hand the memory usage is reduced, on the other hand there is more net
 We recommend using the option with a value high enough to process a large number of notifications, but not so large to crash the process.
 Between 200 and 500 should be fine, and we'll likely process all the notifications in one go.
 
+.. _password-options-label:
+
+Password Options
+----------------
+
+There are three ways to supply a password:
+
+#. Interactively in response to a password prompt.
+#. Sent as a parameter to the command, e.g., ``occ wnd:listen host share username password``.
+#. Read from a file, using the ``--password-file`` switch to specify the file to read from. 
+#. Using 3rd party software to store and fetch the password. When using this option, the 3rd party app needs to show the password as plaintext on standard output.
+
+basic example:
+
+::
+
+   cat /tmp/plainpass | sudo -u www-data ./occ wnd:listen host share username --password-file=-
+
+This provides a bit more security because the /tmp/plainpass password should be owned by root and only root should be able to read the file (0400 permissions), particularly apache shouldn't be able to read it. It's expected that root will be the one to run that command line
+
+Similar as above, but the contents are encoded in base64 format (not much security, but addional obfuscation):
+
+::
+
+   base64 -d /tmp/encodedpass | sudo -u www-data ./occ wnd:listen host share username --password-file=-
+
+Third party password managers can also be integrated. The only requirement is that they have to provide the password in plain text somehow. If not, additional operations might be required to get the password as plain text and inject it in the listener. As an example:
+
+For a more complex test, which might be similar to a real scenario, you can use "pass" as a password manager. You can go through http://xmodulo.com/manage-passwords-command-line-linux.html to setup the keyring for whoever will fetch the password (probably root) and then use something like pass the-password-name | sudo -u www-data ./occ wnd:listen host share username --password-file=-
```

Comment:
then use something like `pass the-password-name | sudo -u www-data ./occ wnd:listen host share username --password-file=-` (that's a code block)